Makefile.am

```diff
@@ -11,18 +11,28 @@ dist_systemdunit_DATA = \
 	eos-live-boot-overlayfs-setup.service \
 	$(NULL)
 
-# Unfortunately, dist chokes on the escaped \x2d, so we need to
-# split this out into EXTRA_DIST unescaped.
+# Unfortunately, the escaped \x2d is handled poorly, so we distribute and
+# install this unit under another name and rename it by hand.
 systemdunit_DATA = \
-	var-endless\\x2dextra.mount
+	var-endless_x2dextra.mount
+
+install-data-hook:
+	pushd "$(DESTDIR)$(systemdunitdir)"; \
+	mv var-endless_x2dextra.mount var-endless\\x2dextra.mount; \
```

Comment:
Also, it might be better to `install` directly instead of implicit `install` + `mv`. I.e, remove the `systemdunit_DATA` setting above so that all automake does is dist the file from the `EXTRA_DIST` below. Then:
```
install-data-hook:
        $(MKDIR_P) $(DESTDIR)$(systemdunitdir)
        $(INSTALL_DATA) $(srcdir)/var-endless_x2dextra.mount $(DESTDIR)$(systemdunitdir)/'var-endless\x2dextra.mount'
```